src/components/helpmenu/helpmenu.html

```diff
@@ -24,4 +24,7 @@
 		<a href="http://dodmrl.com/" ion-button color="dark" id="mrl-deskbook-button">
 			MRL Deskbook
 		</a>
+		<a href="http://dodmrl.com/" ion-button color="dark" id="mrl-resources-button">
```

Comment:
Get clarification on this, two buttons now point towards the same url

Reply:
This is resolved on a different ticket.

Reply:
hmm, I'd suggest we target a staging branch for this - as we need to decouple branches being dependent on each other being merged in to eliminate the duplicate linking.

As an example, by targeting a master branch - once I merge this in, it'll introduce a button that does the same thing as MRL deskbook. But the duplicate links won't be fixed until I merge in both DP3-189 and DP3-188. A solution in this case would be to target a develop or staging branch for all of them and then merge staging into master so the end-user/tester never deals with duplicate buttons.